src/PrestaShopBundle/Resources/config/services/adapter/product.yml

```diff
@@ -225,6 +225,14 @@ services:
       - name: tactician.handler
         command: PrestaShop\PrestaShop\Core\Domain\Product\Command\UpdateProductShippingCommand
 
+  prestashop.adapter.product.command_handler.set_carriers_handler:
```

Comment:
Maybe too early for this blocking PR but we should think of using the new standard in the future ;) https://github.com/PrestaShop/ADR/issues/28